Article_Policy.md

```diff
@@ -0,0 +1,11 @@
+# Policy for Choosing WoT-related Articles
+
+The Web of Things Marketing Task Force collects articles about the Web of Things in their webpage at [docs/about/articles/index.html](docs/about/articles/index.html).
```

Comment:
Not sure but shouldn't it read
**on** their webpage ?

Reply:
It should indeed be "on". Regarding "its" vs "their", I think it comes down to seeing the TF as a group of people (their) or a more abstract thingy (its). I would prefer their in this case.

Alternatively, we can just say on the official web page of W3C Web of Things